sunpy/timeseries/timeseriesbase.py

```diff
@@ -699,3 +767,10 @@ def _parse_file(cls, filepath):
             The path to the file you want to parse.
         """
         raise NoMatchError(f'Could not find any timeseries sources to parse {filepath}')
+
+
+def _figure_to_base64(fig):
```

Comment:
Is this function not used for the map html repr?

Reply:
Change it to be imported from map, maybe can be moved to "utils" but doesnt find a suitable file

Reply:
Putting it in the base utils file seems like a decent idea?